index.html

```diff
@@ -5,8 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
     
+<header>
+    <div class="name logo">
+        <div class="name-text" ><a href="" id="logo">Sungabanja Thawethe</a></div>
+    </div>
+    <nav>
+        <div class="name menuicon">
+        <img class="hamburger" src="Hamburger.png" alt="Hamburger menu icon">
+        </div>
+    </nav>
+</header>
+
+<section class="intro">
+    <div class="headline">
+        <h1 class="hello-me">Hey There. I'm Sunga</h1>
+        <h3 class="SD">I'm a Software Developer</h3>
+        <p class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <img class="icons" src="Social Media.png" alt="Social Media icons">
```

Comment:
- **[OPTIONAL]** The last reviewer didn't ask you but this is a requirement: The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Please 🙏🏽 list the social media icons using `ul` tag for good practices of HTML&CSS.